services/sso/.gitignore

```diff
@@ -0,0 +1,180 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
There are a lot of file types included here that we don't use so do we need to include them?

Reply:
This is just copying and pasting from the template. I think it's probably not a good use of time editing these templates to fit every common service. Happy to discuss.

Reply:
Edited.